src/Symfony/Component/Messenger/Bridge/Amqp/Tests/Transport/ConnectionTest.php

```diff
@@ -506,6 +560,23 @@ public function testObfuscatePasswordInDsn()
         $connection->channel();
     }
 
+    public function testNoCaCertOnSslConnectionFromDsn()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('No CA certificate has been provided. Set "amqp.cacert" in your php.ini or pass the "cacert" parameter in the DSN to use SSL. Alternatively, you can use amqp:// to use without SSL.');
+
+        $factory = new TestAmqpFactory(
+            $amqpConnection = $this->createMock(\AMQPConnection::class),
+            $amqpChannel = $this->createMock(\AMQPChannel::class),
+            $amqpQueue = $this->createMock(\AMQPQueue::class),
+            $amqpExchange = $this->createMock(\AMQPExchange::class)
+        );
+
+        ini_set('amqp.cacert', '');
```

Comment:
That changes a global state, so the current value should be stored and restored after the test.

Reply:
Nice to know, I will modify it :)